### PR TITLE
Style guide feedback thens

### PR DESCRIPTION
**Pull request by pamgoodrich**
Heitor feedback.

### CODE REVIEW

File: 2a.-style-guides/c-create-rules.md [hunk 1]
```diff
@@ -30,9 +32,9 @@ Set severity, name, the message returned during validation, and the description
 
 - **Name**: Provide a short descriptive name that contains no spaces. Example: `require-description`.
 
-- **Message**: Provide the message that will appear in the **API Design Checker** as developers work with APIs. Add message text or include placeholders that are evaluated at runtime:
+- **Message**: Provide the message that will appear in the **API Design Checker** as developers work with APIs. Add text or include placeholders that are evaluated at runtime:
 
-   - `{{error}}`: Returns the function error.
+- `{{error}}`: Returns the function error.
```
Comment (heitortsergent) [inline 2a.-style-guides/c-create-rules.md:37]: This looks like it has the wrong indentation I think.

File: 2a.-style-guides/h-create-custom-functions.md [hunk 2]
```diff
@@ -16,7 +16,9 @@ To create a custom function:
 
 1. Edit a style guide project or an API project that has a style guide enabled.
 2. Select the **Styles** tab.
-3. In the left pane, select the **Add (+)** icon next to **Functions**. (In a Style Guide project, select the **Add (+)** icon from project toolbar, and then select **Add Function**.)
+3. Use one of these options to add a function:
+  - In the left pane from any project, select the **Add (+)** icon next to **Functions**. 
+   - In a Style Guide project, select the **Add (+)** icon from project toolbar, and then select **Add Function**.
```
Comment (heitortsergent) [inline 2a.-style-guides/h-create-custom-functions.md:21]: It's not affecting how it's displayed but looks like there's an extra space here as well.

**Review by heitortsergent — APPROVED** [review]
Just a couple small spacing comments, looks good!

File: 2a.-style-guides/c-create-rules.md (hunk 1 above)
Comment (pamgoodrich) [reply 2a.-style-guides/c-create-rules.md:37]: Fixed

**Pull request merged**